source/extensions/tracers/skywalking/tracer.cc

```diff
@@ -64,6 +64,17 @@ Tracing::SpanPtr Span::spawnChild(const Tracing::Config&, const std::string& nam
   return std::make_unique<Span>(child_span, tracing_context_, parent_tracer_);
 }
 
+void Span::packSpanContextToMetadata(StreamInfo::StreamInfo& info) const {
+  ProtobufWkt::Struct output;
+  auto* fields = output.mutable_fields();
+  (*fields)["sw_service"] = ValueUtil::stringValue(tracing_context_->service());
+  (*fields)["sw_service_instance"] = ValueUtil::stringValue(tracing_context_->serviceInstance());
+  (*fields)["sw_trace_id"] = ValueUtil::stringValue(tracing_context_->traceId());
+  (*fields)["sw_segment_id"] = ValueUtil::stringValue(tracing_context_->traceSegmentId());
+  (*fields)["sw_span_id"] = ValueUtil::numberValue(span_entity_->spanId());
```

Comment:
q: are these fields documented in SW, or just defined here?

Reply:
these values are just defined for envoy. But these values can be used by envoy users. e.g. `DYMAMIC_METADATA("sw_service")` So it must be well documented.